content/blog/2025-03-30-datafusion-python-46.0.0.md

```diff
@@ -0,0 +1,300 @@
+[DataFusion 44.0.0]: https://github.com/apache/datafusion/blob/main/dev/changelog/44.0.0.md
+[String View Pt 1]: https://datafusion.apache.org/blog/2024/09/13/string-view-german-style-strings-part-1/
+[Pt 2]: https://datafusion.apache.org/blog/2024/09/13/string-view-german-style-strings-part-2/
+
+## Coming Soon
+
+There is a lot of excitement around the upcoming work. This list is not comprehensive, but
+a glimpse into some of the upcoming work includes:
+
+- Reusable DataFusion UDFs: The way user defined functions are currently written in
+`datafusion-python` is slightly different from those written for the upstream Rust
+`datafusion`. The core ideas are usually the same, but it means it takes effort for users
+to re-implement functions already written for Rust projects to be usable in Python. Issue
+[#1017] addresses this topic. Work is well underway to make it easier to expose these
+user functions through the FFI boundary. This means that the work that already exists in
+repositories such as those found in the [datafusion-contrib] project can be easily
+re-used in Python. This will provide a low effort way to expose significant functionality
+to the DataFusion in Python community.
+- Additional table providers: We have work well underway to provide a host of table providers
+to `datafusion-python` including: sqlite, duckdb, postgres, odbc, and mysql! In
+[datafusion-contrib #279] we track the progress of this excellent work. Once complete, users
+will be able to `pip install` this library and get easy access to all of these table
+providers. This is another way we are leveraging the FFI work to greatly expand the usability
+of `datafusion-python` with relatively low effort.
+- External catalog and schema providers: For users who wish to go beyond table providers
+and have an entire custom catalog with schema, Issue [#1091] tracks the progress of exposing
+this in Python. With this work, if you have already written a Rust based table catalog you
+will be able to interface it in Python similar to the work described for the table
+providers above.
+
+This is only a sample of the great work that is being done. If there are features you would
+love to see, we encourage you to open an issue and join us as we build something wonderful.
+
+[#1017]: https://github.com/apache/datafusion-python/issues/1017
+[datafusion-contrib #279]: https://github.com/datafusion-contrib/datafusion-table-providers/issues/279
+[#1091]: https://github.com/apache/datafusion-python/issues/1091
+[datafusion-contrib]: https://github.com/datafusion-contrib
+
+## Appreciation
+
+We would like to thank everyone who has helped with these releases through their helpful
+conversations, code review, issue descriptions, and code authoring. We would especially
+like to thank the following authors of PRs who made these releases possible, listed in
+alphabetical order by username: [@chenkovsky], [@CrystalZhou0529], [@ion-elgreco],
```

Comment:
FYI  @chenkovsky, @CrystalZhou0529, @ion-elgreco, @jsai28, @kevinjqliu, @kylebarron, @kosiew, @nirnayroy, and @Spaarsh